devicetypes/gs/gatorsystem-homewatcher.src/gatorsystem-homewatcher.groovy

```diff
@@ -0,0 +1,130 @@
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+metadata {
+    definition (name: "GatorSystem HomeWatcher", namespace: "GS", author: "GS_coder000") {
+    capability "Motion Sensor"
+    capability "Battery"
+    capability "Contact Sensor"
+    capability "Presence Sensor"   
+        
+    // Raw Description 08 0104 0402 00 02 0000 0500 01 0502
+    fingerprint deviceJoinName: "GatorSystem Multipurpose Sensor", manufacturer: "GatorSystem", model: "GSHW01"
```

Comment:
Please place deviceJoinName after manufacturer.

Reply:
I should have asked You in a first place to move deviceJoinName **after a manufacturer and a model.** 
Please, could You do that ?